### PR TITLE
Add the release note for 1.26.4

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

**Review by trustin — APPROVED** [review]
:+1:

File: site/src/pages/release-notes/1.26.4.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Fixed a bug where a <type://RequestLog> isn't complete when an <type://HttpResponse> is cancelled. #5312
+- [gRPC HTTP/JSON transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding) service now sends
+  400 Bad Request when the request body is invalid JSON. #5306 #5307
+
+## 🙇 Thank you
+
+<ThankYou usernames={[
+  'ikhoon',
+  'jrhee17',
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.26.4.mdx:15]: I'd like to give a credit to @ks-yim who reported the bug of #5312 and the reproducer.
```suggestion
  'jrhee17',
  'ks-kim',
```

**Review by ikhoon — APPROVED** [review]
Thanks, @minwoox!

File: site/src/pages/release-notes/1.26.4.mdx (hunk 1 above)
Comment (minwoox) [reply site/src/pages/release-notes/1.26.4.mdx:15]: Added. 😉

File: site/src/pages/release-notes/1.26.4.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+date: 2023-11-2x
```
Comment (jrhee17) [inline site/src/pages/release-notes/1.26.4.mdx:2]: 📓  Just in case you forget to update the date before releasing

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

File: site/src/pages/release-notes/1.26.4.mdx (hunk 2 above)
Comment (minwoox) [inline site/src/pages/release-notes/1.26.4.mdx:2]:
```suggestion
date: 2023-12-07
```

File: site/src/pages/release-notes/1.26.4.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+date: 2023-12-07
```
Comment (minwoox) [inline site/src/pages/release-notes/1.26.4.mdx:2]:
```suggestion
date: 2023-12-08
```

**Pull request merged**